source/gen/stryker-config.yaml

```diff
@@ -0,0 +1,8 @@
+  stryker-config:
```

Comment:
Is it viable to move the `stryker-config.yaml` from `source/gen/` to `source/`?
My argument is that this file is not directly consumed by the `FlashOWare.Generators.csproj`, but rather brining it, the tests and the solution together.
I guess we would need to update the `.sln` and the `.yml` as well.

What do you think?